doc/RESTAPI_Guidelines.md

```diff
@@ -0,0 +1,118 @@
+# Rules for JSON (in NFS) :
```

Comment:
Instead of NFS, I think it should be an NFS control server or management server. What are your thoughts?

Reply:
Done